incipyt/cli.py

```diff
@@ -0,0 +1,46 @@
+"""Command Line Interace functions."""
+
+import click
+import logging
+import pathlib
+
+from incipyt import actions, Environment, process_actions
+
+
+@click.command(help="incipyt is a command-line tool that bootstraps a Python project.")
+@click.argument(
+    "folder",
+    required=True,
+    default=pathlib.Path(),
+    type=click.Path(file_okay=False),
+    callback=lambda _ctx, _param, _path: pathlib.Path(_path),
+)
+@click.version_option()
+@click.option(
+    "--yes",
+    is_flag=True,
+    help="Do not ask confirmation for variables with a default value.",
+)
+@click.option(
+    "--check-build",
+    is_flag=True,
+    help="Build the package after initialization of all files and folders.",
+)
+def main(folder, yes, check_build):
+    logging.basicConfig(level="INFO")
```

Comment:
IMO default level should be WARNING. The verbosity could be increased via cumulative `-v` options and decreased via cumulative `-s` options. Duty for another PR, IMO INFO is fine for now.